WalletWasabi.Fluent/Models/Wallets/TransactionTreeBuilder.cs

```diff
@@ -148,9 +148,9 @@ private TransactionModel CreateRegular(int index, TransactionSummary transaction
 			Type = itemType,
 			Status = status,
 			Confirmations = confirmations,
+			Fee = transactionSummary.GetFee,
```

Comment:
How does this work exactly under the hood? Because the method is passed, `TransactionModel` will keep `TransactionSummary` alive?

Reply:
Oh boy, very good question.

Reply:
Reverted back. Now it should look like on `master`.